Expensify/App
# Alphabetize and shorten pronouns

### Details
Alphabetizes the constants and shortens the strings used for pronouns.

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com/Expensify/Expensify/issues/158312

### Tests

1. Log into app
2. Click on your profile bubble to open up the settings
3. Click **Preferences**
4. Click the **Preferred Pronouns** dropdown and confirm the 4 top options are shortened and in alphabetical order

### QA Steps

Same as tests above:
1. Log into app
2. Click on your profile bubble to open up the settings
3. Click **Preferences**
4. Click the **Preferred Pronouns** dropdown and confirm the 4 top options are shortened and in alphabetical order

### Tested On


### Screenshots


#### Web

![image](https://user-images.githubusercontent.com/3387421/113034290-c859d200-9146-11eb-8f36-cae9b2670994.png)
